randobot/zsr.py

```diff
@@ -144,3 +146,17 @@ def get_hash(self, seed_id):
             self.hash_map.get(item, item)
             for item in settings['file_hash']
         )
+    
+    def load_qualifier_placements(self):
+        placement = requests.get(self.qualifier_placement_endpoint).json()
+        return placement
+    
+    def load_draftable_settings(self):
+        settings = requests.get(self.settings_dev_endpoint).json()
+        available_settings = [name for name in settings['S6 Tournament'].keys() if name in (
```

Comment:
Should this be S7?

Reply:
It technically doesn't exist yet so I was just using this is a testing point but will change once a preset for s7 is live